apps/browser-extension-wallet/src/hooks/useGetHandles.ts

```diff
@@ -0,0 +1,11 @@
+import { useWalletStore } from '@src/stores';
+import { isAdaHandleEnabled } from '@src/features/ada-handle/config';
+import { useObservable } from './useObservable';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const useGetHandles = () => {
```

Comment:
Maybe we should add a return type and remove the disable eslint lint comment

```suggestion
export const useGetHandles = (): HandleInfo[] | undefined => {
```